includes/class-sensei-settings-api.php

```diff
@@ -409,6 +448,10 @@ private function render_additional_section_elements( $section_id ) {
 		if ( 'woocommerce-settings' === $section_id && ! apply_filters( 'sensei_settings_woocommerce_hide', false ) ) {
 			$this->render_woocommerce_upgrade_settings();
 		}
+
+		if ( 'sensei-content-drip-settings' === $section_id && ! apply_filters( 'sensei_settings_content_drip_hide', false ) ) {
```

Comment:
Is there a description for this hook somewhere?

Reply:
Hey I've added description on 2 places where is it used and also did the same thing for woocommerce change (I forgot in previous pr):
Content drip: 
[First](https://github.com/Automattic/sensei/pull/4822/commits/942024b179aa7d6ed5a0cfad3466362ee0774576#diff-7c8b7a09d61ea383f0f202b51c25d8cdbc4be93ce5072b06d1065c42b04290f4R462-R470) and [Second](https://github.com/Automattic/sensei/pull/4822/commits/942024b179aa7d6ed5a0cfad3466362ee0774576#diff-6fc0c94d5794cb7f95469bdef471c292a5d553a8a4664d4b48633a2e7449261dR183-R191) 

Woocommerce: 
[First](https://github.com/Automattic/sensei/pull/4822/commits/942024b179aa7d6ed5a0cfad3466362ee0774576#diff-7c8b7a09d61ea383f0f202b51c25d8cdbc4be93ce5072b06d1065c42b04290f4R448-R456) and [Second](https://github.com/Automattic/sensei/pull/4822/commits/942024b179aa7d6ed5a0cfad3466362ee0774576#diff-7c8b7a09d61ea383f0f202b51c25d8cdbc4be93ce5072b06d1065c42b04290f4R448-R456)